Signal/src/ViewControllers/MessagesViewController.m

```diff
@@ -3158,7 +3162,7 @@ - (void)endRecordingVoiceMemo
     self.audioRecorder = nil;
 
     NSString *filename = [NSLocalizedString(@"VOICE_MESSAGE_FILE_NAME", @"Filename for voice messages.")
-        stringByAppendingPathExtension:[MIMETypeUtil fileExtensionForUTIType:(NSString *)kUTTypeMPEG4Audio]];
+        stringByAppendingPathExtension:@".m4a"];
```

Comment:
Ah, this is what https://github.com/WhisperSystems/SignalServiceKit/pull/206 is about, right?